staging/publishing/rules.yaml

```diff
@@ -322,4 +322,13 @@ rules:
   - source:
       branch: master
       dir: staging/src/k8s.io/cri-api
+    dependencies:
```

Comment:
Can you move `name: master` above dependencies?

Sorry for not pointing this out earlier, but you could also add import restrictions for cri-api here: https://github.com/kubernetes/kubernetes/blob/master/staging/publishing/import-restrictions.yaml? Thanks!

Reply:
Move the `name: master`
Added import restrictions based on https://github.com/alok87/kubernetes/blob/dep-streaming-moved-out/staging/src/k8s.io/cri-api/go.mod#L15-L20

Why do we specify import restrictions? What does the bot do reading this file?

Reply:
> Added import restrictions based on https://github.com/alok87/kubernetes/blob/dep-streaming-moved-out/staging/src/k8s.io/cri-api/go.mod#L15-L20

Ah, I meant adding the import restrictions to this file: https://github.com/kubernetes/kubernetes/blob/master/staging/publishing/import-restrictions.yaml.

> Why do we specify import restrictions? What does the bot do reading this file?

It makes sure that a package won't import any additional packages accidentally. Importing a new package would mean explicitly adding it in the list in that file.

The main significance of that file is to make sure that a staging repo won't import k/k though...

Reply:
Got it. thanks
I have added it in `import-restrictions.yaml` only

Reply:
@alok87 I can't see the change in the diff. Maybe forgot to push? :upside_down_face:

Reply:
please check now :)